docs/internals/contributing/writing-code/coding-style.txt

```diff
@@ -89,6 +124,9 @@ Python style
         """
         ...
 
+
+
+
```

Comment:
```suggestion
```

(Huh. The suggestion only shows removal of one line, not three.)

Reply:
I was gonna put links there. Thanks!